cartoframes/assets/templates/viz/widgets/default.html.j2

```diff
@@ -0,0 +1,18 @@
+<as-widget-header
+  header="{{widget.title}}"
+  subheader="{{widget.description}}">
+</as-widget-header>
+
+{% if widget.name %}
+  <p class="as-title as-m--0 as-p--0 as-font--medium" id="{{id}}">
+    <span class="as-loading as-loading--s">
+      <svg viewBox="0 0 50 50">
+        <circle cx="25" cy="25" r="20" fill="none" />
+      </svg>
+    </span>
+  </p>
+{% endif %}
+
+{% if widget.footer %}
```

Comment:
By default, Airship widgets don't contain a `footer` or `source` section, this has been added for the non-bridge widgets. In order to homogenize the Widgets in CF it maybe makes sense to add only title and description. What do you think?

Reply:
Ok, this makes sense to me, but this is one of the tasks I'd move to the review widgets task, let's add it there 👌